include/ipmitool/ipmi_delloem.h

```diff
@@ -344,7 +344,7 @@ typedef struct _power_headroom
 } __attribute__ ((packed)) POWER_HEADROOM;
 
 struct vFlashstr {
-	uint8_t val;
+	uint32_t val;
```

Comment:
It's actually unclear why a custom `struct vFlashstr` is used instead of generic `struct valstr` from `helper.h`.
The function `get_vFlash_compcode_str()` can as well be ditched and the generic `val2str()` can be used instead.
I think it's a good moment to get rid of that de-facto duplicate code.

Reply:
I agree. I will commit it later